vulcanai/models/metrics.py

```diff
@@ -45,11 +45,12 @@ def get_score(targets, predictions, metrics='accuracy', average=None,
             metrics: list of strings
                 The strings definition the "get_"... functions to call
             average: string,
-                [None, ‘binary’ (default), ‘micro’, ‘macro’, ‘samples’, ‘weighted’]
+                [None, ‘binary’ (default), ‘micro’, ‘macro’, ‘samples’,
+                ‘weighted’]
                 This parameter is required for multiclass/multilabel targets.
                 If None, the scores for each class are returned.
-                Otherwise, this determines the type of averaging performed on the
-                data.See scikit learn
+                Otherwise, this determines the type of averaging performed on
+                the data.See scikit learn
```

Comment:
should we provide a link or nah?

Reply:
I provided further explanation - it's all documented on the individual metric functions so there's not one link.